validations/task.js

```diff
@@ -0,0 +1,112 @@
+const express = require("express");
+const mongoose = require("mongoose");
+
+function validateID(id) {
```

Comment:
if this one is going to be used as a middleware, it has to be a middleware (using next, etc)